src/Entities/BankAccount.php

```diff
@@ -122,6 +122,14 @@ public function setCustomFields($value)
         return $this->setAttribute('customFields', $value);
     }
 
+    /**
+     * @return string
+     */
+    public function getRoutingNumber()
```

Comment:
Looks like this property is missing in changelog

Reply:
There is setter defined, so it's no new property. Updated changelog.